Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashC.h

```diff
@@ -93,6 +93,8 @@ void bsg_kscrash_reinstall(const char *const crashReportFilePath,
                            const char *const crashID);
 
 /** Set whether or not to print a stack trace to stdout when a crash occurs.
+ *
+ * Note: This has been removed, and now does nothing.
```

Comment:
Why not remove `bsg_kscrash_setPrintTraceToStdout()` and `config.printTraceToStdout`?

Reply:
Oh lol Nick beat me to it!